### DISCUSSION TITLE
creating a container block

### DISCUSSION THREAD

**Discussion by aviadn** [jeffreyvr/dropblockeditor#29] [General]
Hey, thanks for this great package. I'm looking for ways to create some kind of container block, mainly for layout - add the ability to inline 2 blocks or more, or use grid or flex on a group of blocks. Any suggestions on how to approach this?

**Comment by jeffreyvr** [comment]
Thank you! Yes this would be very cool, but it's not possible (yet) - unfortunately. I have an idea how to achieve it but it will be a more complex feature to add, and I need to find some time to build it :)

**Reply by arnor01** [reply]
If there's anything I can do to help you with it, please let me know! Would love to see that feature. Tried to implement it in the package myself, but without much success. :sweat_smile:

**Reply by Josh20022** [reply]
@jeffreyvr I am going to implement this feature (will try within next month), if you can share your ideas would be helpful. i will share code after completion
